src/vcf/store_parse_policy.cpp

```diff
@@ -166,6 +166,8 @@ namespace ebi
             throw new PositionBodyError{state.n_lines};
         }
 
+        check_sorted(state, position);
```

Comment:
Should errors due to unsorted records (chrom+pos related) be reported before semantic chrom errors? Maybe both the sorting and duplicates checks could go into ParserImpl. I think we already discussed moving the duplicates at least?